packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -547,11 +553,14 @@ describe("Garbage Collection Tests", () => {
                 clock.tick(1);
                 await updateAllNodesAndRunGC(garbageCollector);
                 const expectedEvents: Omit<ITelemetryBaseEvent, "category">[] = [];
-                if (deleteEventName) {
+
+                if (expectDeleteLogs) {
                     expectedEvents.push(
                         { eventName: deleteEventName, timeout, id: nodes[2] },
                         { eventName: deleteEventName, timeout, id: nodes[3] },
                     );
+                } else {
+                    assert(!mockLogger.events.some((event) => event.eventName === deleteEventName), "Should not have any delete events logged");
```

Comment:
Nice!

Reply:
Thanks. After the fact wasn't sure it was worth it, so glad you like it :)  Also it would be cool if mockLogger was more expressive and you could tell it about events you don't want as well.